lib/appium_lib/ios/xcuitest/command/gestures.rb

```diff
@@ -121,10 +121,10 @@ def two_finger_tap(element:)
         #   Otherwise they should be calculated relatively to screen borders.
         #
         #   ```ruby
-        #   tap x: 100, y: 100
-        #   tap x: 100, y: 100, element: find_element(:accessibility_id, "some item")
+        #   one_finger_tap x: 100, y: 100
+        #   one_finger_tap x: 100, y: 100, element: find_element(:accessibility_id, "some item")
         #   ```
-        def tap(x:, y:, element: nil)
+        def one_finger_tap(x:, y:, element: nil)
```

Comment:
renamed here since Ruby already has `tap`. This change was addressed in changelog
https://apidock.com/ruby/Object/tap